test/fixtures/images.md

```diff
@@ -0,0 +1,31 @@
+# Image references
+
+<!-- Valid: -->
+
+Relative ![image](./examples/image.jpg)
```

Comment:
Can you also put an example that shows omitting the `./` just to make it obvious that it's supported as well?

Reply:
done!